salt/beacons/__init__.py

```diff
@@ -129,6 +129,25 @@ def add_beacon(self, name, beacon_data):
 
         return True
 
+    def modify_beacon(self, name, beacon_data):
+        '''
+        Modify a beacon item
```

Comment:
Can you flesh out the docs a bit here by including definitions for `name` and `beacon_data`?

Can you also add a `.. versionadded:: Beryllium` tag here?